src/main/java/propertyFiles/Bundle.properties

```
@@ -1209,6 +1209,7 @@ dataset.exportBtn.itemLabel.datacite=DataCite
 dataset.exportBtn.itemLabel.json=JSON
 dataset.exportBtn.itemLabel.oai_ore=OAI_ORE
 dataset.exportBtn.itemLabel.dataciteOpenAIRE=OpenAIRE
+dataset.exportBtn.itemLabel.html=DDI HTML codebook
```

Title case is used for buttons, like this:

```suggestion
dataset.exportBtn.itemLabel.html=DDI HTML Codebook
```